docs/shared/angular-plugin.md

```diff
@@ -26,6 +28,7 @@ npm install -D @nrwl/angular
 
 Building Angular applications within an Nx workspace is similar to building within a vanilla Angular CLI workspace, with a few differences.
 
+- ESLint is used as the default linter, configurable in the generators section of the workspace configuration.
 - Jest is used as the default unit test runner, configurable in the generators section of the workspace configuration file.
```

Comment:
I know this wasn't your doing, but I feel like these three bullet points should have the same wording. The second one being slightly off is strange.